docs/releases/v2022.1.rst

```diff
@@ -0,0 +1,357 @@
+``ath79-nand`` target respectively.
+
+Missing devices
+~~~~~~~~~~~~~~~
+
+The following devices have not yet been integrated into Gluons ath79
+targets.
+
+-  8Devices
+
+   -  Carambola 2
+
+-  Aerohive
+
+   -  HiveAP 121
+
+-  Allnet
+
+   -  ALL0315
+
+-  Buffalo
+
+   -  WZR-HP-G300NH2
+   -  WZR-HP-G450H
+
+-  GL.iNet
+
+   -  6408A v1
+
+-  NETGEAR
+
+   -  WNDR3800
+   -  WNDR4300
+   -  WNDRMAC
+   -  WNDRMAC v2
+
+-  OCEDO
+
+   -  KOALA
+
+-  TP-Link
+
+   -  WR2543
+
+-  WD
+
+   -  MyNet N600
+   -  MyNet N750
+
+-  ZyXEL
+
+   -  NB6616
+   -  NB6716
+
+Features
+--------
+
+Wireguard
+~~~~~~~~~
+
+Gluon got WireGuard support. This allows offloading **encrypted**
+connections into kernel space, increasing performance by forwarding
+packets without the need for context switches between user and kernel
+space.
+
+In order to reuse existing (already verified) fastd-keypairs for
+WireGuard, a key derivation procedure is currently being developed
+(`currently being
+developed <https://github.com/freifunk-gluon/gluon/pull/2601>`__). This
+should ease migration from fastd to WireGuard in case whitelisting VPN
+keys is desired.
+
+fastd L2TP
+~~~~~~~~~~
+
+fastd can now act as a connection broker for unencrypted L2TP-based
+tunneling within Gluons mesh-vpn framework.
+
+In addition to a sufficient configured fastd-based VPN server, this
+requires adding the ``null@l2tp`` method to the fastd site.
+
+Major changes
+-------------
+
+Network changes (DSA / Upgrade-Behavior)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ramips-mt7621 and lantiq-xrx200 targets now use the upstream DSA
+subsystem instead of OpenWrt swconfig for managing ethernet switches.
+
+Gluon detects the existing user-intent and automatically applies it over
+to DSA syntax. See the section about network reconfiguration for more
+details.
+
+System reconfiguration
+~~~~~~~~~~~~~~~~~~~~~~
+
+The network and system-LED configurations are now re-generated after
+each update / invocation of ``gluon-reconfigure``.
+
+The user-intent is preserved within Gluon’s implemented functionality
+(Wired-Mesh / Client access / WAN).
+
+As an additional feature, Gluon now supports assigning roles to
+interfaces. This behavior is explained here –>
+https://github.com/freifunk-gluon/gluon/pull/2584/files
+
+Site changes
+------------
+
+VPN provider MTU
+~~~~~~~~~~~~~~~~
+
+To account for multiple VPN methods available for a site, the MTU used
+for the VPN tunnel connection is now moved to the specific VPN provider
+configuration. For fastd this means that ``mesh_vpn.mtu`` needs to be
+moved to ``mesh_vpn.fastd.mtu``. (#2352)
+
+Minor changes
+-------------
+
+-  The ``brcm2708-bcm2708`` ``brcm2708-bcm2709`` ``brcm2708-bcm2710``
+   targets were renamed to ``bcm27xx-bcm2708`` ``bcm27xx-bcm2709`` and
+   ``bcm27xx-bcm2710``.
+-  The GL.iNet GL-AR750S was moved to the ``ath79-nand`` subtarget
+-  Gluon now ships the ath10k-ct firmware deriavation for ath10k Wave 2
```

Comment:
"deriavation" has a spelling error. and wouldn't "variant" be a better word here?

Reply:
Expanded to full list of affected chipsets